src/lib/blog/util.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Takes a string (either html or plaintext) and calculates the time
+ * (in minutes) to read a blog post, By default it uses 225 the default adult
+ * average reading time based on a quick google search, but may be overrided
+ * with the wpm parameter
+ * @param string The string containing the blog text
+ * @param wpm (Optional) defines words per minute to assume the reader reads at
+ * @returns The estimated read time of an article in minutes
+ */
+export function readingTime(string: string, wpm = 225) {
```

Comment:
Nit: I would avoid calling the name of the variable the same thing as a primitive type. The argument named `string` could  instead be named `content`, `corpus`, `text`, etc.